src/CommunityToolkit.Maui/Alerts/Snackbar/SnackBar.shared.cs

```diff
@@ -85,7 +85,7 @@ public Snackbar()
 		};
 	}
 
-#if NET6_0
+#if !(IOS || ANDROID || MACCATALYST || WINDOWS)
```

Comment:
Let's remove net6.0 from the target framework. In that case we do not need IF

Reply:
If we do this we will break our unit test :(

Reply:
Hmm, is it required to be net6.0 for test project?
Can it be net6.0-platform?